Framework/PythonInterface/plugins/algorithms/SaveHKLCW.py

```diff
@@ -0,0 +1,96 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2020 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+# SPDX - License - Identifier: GPL - 3.0 +
+
+import numpy as np
+from mantid.api import (AlgorithmFactory, PythonAlgorithm, FileAction, FileProperty,
+                        IPeaksWorkspaceProperty)
+from mantid.kernel import Direction, logger
+
+
+class SaveHKLCW(PythonAlgorithm):
+    def category(self):
+        return 'Diffraction\\DataHandling'
+
+    def summary(self):
+        """
+        summary of the algorithm
+        :return:
+        """
+        return "Save a PeaksWorkspace to SHELX76 format required by WINGX"
+
+    def name(self):
+        return "SaveHKLCW"
+
+    def seeAlso(self):
+        return ["SaveHKL"]
+
+    def PyInit(self):
+        self.declareProperty(IPeaksWorkspaceProperty('Workspace', '', direction=Direction.Input),
+                             doc='PeaksWorkspace that should be used.')
+        self.declareProperty(FileProperty('OutputFile',
+                                          '',
+                                          action=FileAction.Save,
+                                          extensions=['hkl', 'int'],
+                                          direction=Direction.Input),
+                             doc='Output File.')
+        self.declareProperty('Header',
+                             True,
+                             direction=Direction.Input,
+                             doc='If to include the header in the output')
+        self.declareProperty('DirectionCosines',
+                             False,
+                             direction=Direction.Input,
+                             doc='If to include the direction cosines output')
+
+    def PyExec(self):
+
+        peak_ws = self.getProperty('Workspace').value
+        outFile = self.getPropertyValue('OutputFile')
+        header = self.getProperty('Header').value
+        directionCosines = self.getProperty('DirectionCosines').value
+
+        with open(outFile, 'w') as f:
+            if header:
+                f.write("Single crystal data\n")
+                if directionCosines:
+                    f.write("(3i4,2f8.2,i4,6f8.5)\n")
+                else:
+                    f.write("(3i4,2f8.2,i4)\n")
+                wavelengths = [p.getWavelength() for p in peak_ws]
+                print(np.std(wavelengths))
```

Comment:
Is this print statement needed? this is what is causing your doctest to fail.

Reply:
I thought I had removed that, the doc-tests fail otherwise.